deltachat-ios/Controller/AccountSetupController.swift

```diff
@@ -102,9 +100,9 @@ class AccountSetupController: UITableViewController, ProgressAlertHandler {
         return cell
     }()
 
-    private lazy var restoreCell: UITableViewCell = {
-        let cell = UITableViewCell(style: .value1, reuseIdentifier: nil)
-        cell.textLabel?.text = String.localized("import_backup_title")
+    private lazy var restoreCell: BasicCell = {
```

Comment:
All normal UITableViewCell have been replaced by a custom cell (BasicCell) that resizes properly if the user changes the system font size.

Reply:
replacing `UITableViewCell.textLabel` by `BasicCell.title.text` removes knowledge about the _semantic_ of that field. same for replacing `detailTextLabel` by `value.text` and maybe other occurances. 

this remove of _semantic_ may introduce additional/new accessibility issues as readers etc. does not have an idea about what the custom text fields are.

also, already now, the change introduce a visual difference (different gray levels), so that our settings appear different from the system ones.

also, if with some os update, displays things differently, we would need to adapt our BasicCell.

finally, i just checked that, the `UITableViewCell.textLabel` and `detailTextLabel` already use adaptive-font sizes, so there is no need to change that.

Reply:
@r10s unfortunately UITableViewCell.textlabel and detailTextLabel are performing very poorly on iOS 13.5 wrt adaptive font-sizes which is why I introduced BasicCell. Just try to switch between scaling up and down the font sizes several times and switch in between to the app with a view containing UITableViewCells. The layout breaks.

Reply:
I think the point about the accessibility issues are a very good point, indeed, which needs to be fixed. I'll checkout the behavior of the screen reader for BasicCells. 
Wrt the colors, we _can_ check if there are system colors we use. However, we use custom colors all over the place. Also the behavior of BasicCells already differs from normal UITableViewCells (2 line layout for huge fonts). So it's not ment to be a default system cell.